src/src/rx_main.cpp

```diff
@@ -1225,6 +1241,31 @@ void loop()
     }
     #endif
 
+    if (config.IsModified())
+    {
+        uint32_t start = micros();
+        hwTimer.stop();
+        Radio.Pause();
+        config.Commit();
+        Radio.Resume();
+        /*
+        // Currently this code is not aligining the FHSSptr correctly so it times out and get LostConnection
+        while((micros() - start) % ExpressLRS_currAirRate_Modparams->interval > 50) ;
+        uint32_t offset = micros() - start;
+        int intervals = offset / ExpressLRS_currAirRate_Modparams->interval - 1;
+        NonceRX = (NonceRX + intervals) % ExpressLRS_currAirRate_Modparams->FHSShopInterval;
+        FHSSptr = (FHSSptr + (intervals / ExpressLRS_currAirRate_Modparams->FHSShopInterval)) % FHSS_SEQUENCE_CNT;
+        Radio.SetFrequencyReg(FHSSgetNextFreq());
+        */
```

Comment:
@CapnBry can you figure out why this doesn't work? Does my math just suck that hard!